src/main/kotlin/NoticeList.kt

```diff
@@ -0,0 +1,31 @@
+/* Этот класс содержит список предупреждений, которые выводятся юзеру, когда он что-то сделал не так*/
+class NoticeList {
+    companion object {
```

Comment:
```suggestion
object NoticeList {
```
Если класс состоит только из `companion object`, то его целиком можно определить как `object`